junwoo/controllers/postController.js

```diff
@@ -0,0 +1,52 @@
+//controller/postController.js
+
+const postService = require("../services/postService");
+
+const postUp = async (req, res) => {
+  try {
+    const { title, content, image, user_id } = req.body;
+    if (!title || !content || !image || !user_id) {
+      return res.status(400).json({ message: "KEY_ERROR" });
+    }
+
+    await postService.postUp(title, content, image, user_id);
+
+    res.status(201).json({ message: "POSTUP_SUCCESS" });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.status || 500).json({ message: err.message });
+  }
+};
+
+const usersPost = async (req, res) => {
+  const { userId } = req.params;
+  const posts = await postService.usersPost(userId); // == [{}, {}, {}]
```

Comment:
불필요한 코드 혹은 주석은 삭제해주세요!
```suggestion
  const posts = await postService.usersPost(userId);
```